libfreerdp/primitives/primitives.c

```diff
@@ -165,6 +165,7 @@ static primitives_YUV_benchmark* primitives_YUV_benchmark_init(primitives_YUV_be
 	return ret;
 }
 
+#if defined(HAVE_CPU_OPTIMIZED_PRIMITIVES) && defined(WITH_OPENCL)
```

Comment:
Shouldn't it be a `or` instead of a `and` ?

Reply:
no, `and` because the benchmark is only run if we have more than one optimized primitive implementation.